consensus/block_validation.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/filecoin-project/go-filecoin/plumbing/clock"
```

Comment:
I think this is the first time plumbing has made its way into the consensus package.  I am interested in the long term implications of this change.  Does it mean consensus can't be part of plumbing itself?  Does it mean consensus will have a stronger pull towards being put on plumbing?

It seems like there is lack of intentional design around the interactions between "core components" and the plumbing injection things.  These are just general observations, no action required in this PR.